cas/st/ctgdump_tesh.sh

```diff
@@ -63,7 +63,7 @@ kvs_create_n_insert()
 		echo "m0kv failed"
 	}
 
-	for ((j=0; j<$num_of_kv; j++)); do
+	for ((j=0; j<num_of_kv; j++)); do
```

Comment:
$var is not necessary in $(()), but it is perfectly valid. The advantage of always using $var is that you do not introduce bugs by accidentally copying an expression from within $(()).

Reply:
Agree. I found this rule of Codacy is annoying.
$var is more readable, and less error-prone. 
If possible, I'd suggest to disable this rule.

Reply:
Are all changes in this PR for this same pattern? If so, seems like we shouldn't have created the issue in the first place and instead we should have configured Codacy to reflect our coding style.

Reply:
Agree with @johnbent ... who owns updating Codacy rules/configuration?

Reply:
From every scrum there are people who have been given right to disable/ignore the issue. Currently I have disabled this pattern so that it wont give warnings for this pattern in future.